cli/run/loadConfigFile.ts

```diff
@@ -13,7 +14,11 @@ import batchWarnings, { BatchWarnings } from './batchWarnings';
 import { addCommandPluginsToInputOptions } from './commandPlugins';
 
 function supportsNativeESM() {
-	return Number(/^v(\d+)/.exec(process.version)![1]) >= 13;
+	const version = process.version.match(/^v(\d+)\.(\d+)\.\d+$/);
+	const major = parseInt(version[1], 10);
+	const minor = parseInt(version[2], 10);
+
+	return major >= 14 || (major === 13 && minor >= 2) || (major === 12 && minor >= 17);
```

Comment:
That is definitely a reasonable improvement that I would merge even without a major release.